styles/styles.scss

```diff
@@ -83,6 +83,16 @@ article a {
         color: $white;
       }
     }
+
+    // For Navigation Menu in light mode
+    .container.has-side-nav {
```

Comment:
This is already inside the `.has-side-nav` selector, so the SCSS nesting looks incorrect

Reply:
@nschonni : Thanks for pointing it out, and a quick fix for it :)